Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.6
+
+# Dependencies
+RUN go get github.com/gorilla/securecookie && \
+    go get github.com/gorilla/sessions && \
+    go get github.com/hoisie/mustache && \
+    go get github.com/unixpickle/f1le
+
+# Application install
+RUN go install github.com/unixpickle/f1le
```

Comment:
This could be done fairly easily. Replace the "Dependencies" and "Application install" sections with this:

```
# Application install
ADD . /go/src/f1le
RUN go get f1le
RUN go install f1le
```